frontend/components/Header.js

```diff
@@ -337,7 +337,9 @@ class Header extends Component {
                         <a href=".">Peer worker programs</a>
                       </li>
                       <li>
-                        <a href=".">Outreach</a>
+                        <Link href="/outreach">
+                          <a>Outreach</a>
+                        </Link>
```

Comment:
I just realized something, I have to also fix my links. At the moment, this link will only work for mobile, but not for tablets and desktop. So we have to fix all the other pages to make it consistent